Modules/Remote/IOMeshSWC.remote.cmake

```diff
@@ -49,3 +49,4 @@ itk_fetch_module(
   GIT_REPOSITORY https://github.com/InsightSoftwareConsortium/ITKIOMeshSWC.git
   GIT_TAG 6375295e7802c6a064c90d11f48b3ea238ac8d76
   )
+mark_as_advanced(FORCE Module_IOMeshSWC)
```

Comment:
Why mark this as advanced, while most other remote modes are not? And why is `FORCE` needed?

Reply:
I am not sure why it is needed here.